best-practices/documentation/README.md

```diff
@@ -7,4 +7,34 @@ Ruby projects should use [YARD](http://yardoc.org/) for documentation purposes.
 
 One benefit is YARD runs a live documentation server for the community, hosting docs for all RubyGems and Github projects. The project is at [RubyDoc.info](http://www.rubydoc.info/) (formerly rdoc.info)
 
-A [useful Yardoc cheatsheet](https://gist.github.com/chetan/1827484)
\ No newline at end of file
+A [useful Yardoc cheatsheet](https://gist.github.com/chetan/1827484)
+
+## HTTP API
+
+If your project exposes an HTTP API, you should consider documenting the API using the [OpenAPI Specification](https://swagger.io/docs/specification/about/) (formerly known as Swagger Specification). The OpenAPI Specification allows for rich description of HTTP APIs&mdash;paths, parameters, responses, headers, status codes, etc.&mdash;in machine-readable formats. 
+
+### Specification Formats
+
+OpenAPI Specifications may be expressed as either YAML or JSON. Thus far, we have preferred YAML over JSON because it allows commenting and, unlike JSON, does not care about e.g., trailing commas.
+
+### API Documentation Examples
+
+* https://sul-dlss.github.io/technical-metadata-service/
+* https://sul-dlss.github.io/sdr-api/
+* https://sul-dlss.github.io/preservation_catalog/
+* https://sul-dlss.github.io/dor-services-app/
+* https://sul-dlss.github.io/suri-rails/
+* https://sul-dlss.github.io/cocina-models/
+* https://sul-dlss.github.io/workflow-server-rails/ (in progress)
+
+### How To Add OpenAPI To A Project
+
+Here are the steps we have been following:
+
+1. Create `openapi.yml`: feel free to copy from an existing project, paste, and adjust ([example](https://github.com/sul-dlss/technical-metadata-service/blob/master/openapi.yml))
+1. Validate `openapi.yml` in the CI build ([CircleCI example](https://github.com/sul-dlss/technical-metadata-service/blob/6c8151b1ca713061c227e8030f03c6531eee1093/.circleci/config.yml#L103-L114))
+1. Add OpenAPI validation badge to the README ([example](https://github.com/sul-dlss/technical-metadata-service/blob/6c8151b1ca713061c227e8030f03c6531eee1093/README.md#L5))
+1. Add Committee gem and enable request validation (ideally, also add  response validation; if you don't, add the commented-out response validation code and write up a new issue), then fix all the stuff that breaks ([example](https://github.com/sul-dlss/technical-metadata-service/blob/6c8151b1ca713061c227e8030f03c6531eee1093/config/application.rb#L47-L52))
+1. Create `docs/` folder: copy artifacts from another repo, *e.g.*, technical-metadata-service ([example](https://github.com/sul-dlss/technical-metadata-service/tree/6c8151b1ca713061c227e8030f03c6531eee1093/docs))
```

Comment:
"copy artifacts from another repo" -- so these are config files???  And they would apply to any repo?

Reply:
Re-wording.